powershellgroup/powershellgroup.resource.ps1

```diff
@@ -33,7 +33,7 @@ function RefreshCache
     }
 }
 
-if (($PSVersionTable.PSVersion.Major -ge 7) -and ($PSVersionTable.PSVersion.Minor -ge 4) `
+if (($PSVersionTable.PSVersion.Major -ge 7) -and ($PSVersionTable.PSVersion.Minor -eq 4) `
```

Comment:
Looks like "Major -ge 7)" should also be "-eq",
otherwise this will be triggered for 8.4-previews as well... and 9.4-previews... etc...

Reply:
true